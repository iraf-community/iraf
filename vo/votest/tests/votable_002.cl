#
#  Print VOTable column information

# Set the test description string.
votest.descr = "TABLES support: Print VOTable column info"

# Convert the data$logical to a local path.
s1 = data_url // "/usno-b.xml"
s2 = "file://" // data_path // "/usno-b.xml"
s3 = "file:///localhost" // data_path // "/usno-b.xml"


fcache init

# Execute the test commands.
print ("\nLogical Path:  data$usno-b.xml")		# logical path
tlcol ("data$usno-b.xml")

print ("\nHTTP URI:  " // s1)				# remote http URI
tlcol (s1)

print ("\nFile URI:  " // s2)				# file URI
tlcol (s2)

print ("\nFile URI:  " // s3)				# file URI
tlcol (s3)
