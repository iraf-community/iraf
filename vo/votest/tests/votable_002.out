
Logical Path:  data$usno-b.xml
# data$usno-b.xml
id               CH*13       %-13s ""              
RA               D         %25.16g deg             
DEC              D         %25.16g deg             
sRa              I            %11d mas             
sDec             I            %11d mas             
sRaEp            I            %11d mas             
sDeEp            I            %11d mas             
MuRA             D         %25.16g mas/yr          
MuDEC            D         %25.16g mas/yr          
sMuRA            D         %25.16g mas/yr          
sMuDE            D         %25.16g mas/yr          
B1               D         %25.16g mag             
R1               D         %25.16g mag             
B2               D         %25.16g mag             
R2               D         %25.16g mag             
I2               D         %25.16g mag             

HTTP URI:  http://iraf.noao.edu/votest//usno-b.xml
# http://iraf.noao.edu/votest//usno-b.xml
id               CH*13       %-13s ""              
RA               D         %25.16g deg             
DEC              D         %25.16g deg             
sRa              I            %11d mas             
sDec             I            %11d mas             
sRaEp            I            %11d mas             
sDeEp            I            %11d mas             
MuRA             D         %25.16g mas/yr          
MuDEC            D         %25.16g mas/yr          
sMuRA            D         %25.16g mas/yr          
sMuDE            D         %25.16g mas/yr          
B1               D         %25.16g mag             
R1               D         %25.16g mag             
B2               D         %25.16g mag             
R2               D         %25.16g mag             
I2               D         %25.16g mag             

File URI:  file:///iraf/iraf/vo/votest/data//usno-b.xml
# file:///iraf/iraf/vo/votest/data//usno-b.xml
id               CH*13       %-13s ""              
RA               D         %25.16g deg             
DEC              D         %25.16g deg             
sRa              I            %11d mas             
sDec             I            %11d mas             
sRaEp            I            %11d mas             
sDeEp            I            %11d mas             
MuRA             D         %25.16g mas/yr          
MuDEC            D         %25.16g mas/yr          
sMuRA            D         %25.16g mas/yr          
sMuDE            D         %25.16g mas/yr          
B1               D         %25.16g mag             
R1               D         %25.16g mag             
B2               D         %25.16g mag             
R2               D         %25.16g mag             
I2               D         %25.16g mag             

File URI:  file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml
# file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml
id               CH*13       %-13s ""              
RA               D         %25.16g deg             
DEC              D         %25.16g deg             
sRa              I            %11d mas             
sDec             I            %11d mas             
sRaEp            I            %11d mas             
sDeEp            I            %11d mas             
MuRA             D         %25.16g mas/yr          
MuDEC            D         %25.16g mas/yr          
sMuRA            D         %25.16g mas/yr          
sMuDE            D         %25.16g mas/yr          
B1               D         %25.16g mag             
R1               D         %25.16g mag             
B2               D         %25.16g mag             
R2               D         %25.16g mag             
I2               D         %25.16g mag             
